x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/hooks/use_create_analytics_form/use_create_analytics_form.ts

```diff
@@ -130,19 +131,21 @@ export const useCreateAnalyticsForm = (): CreateAnalyticsFormProps => {
     const indexPatternName = destinationIndex;
 
     try {
-      const newIndexPattern = await mlContext.indexPatterns.make();
-
-      Object.assign(newIndexPattern, {
-        id: '',
+      await mlContext.indexPatterns.createAndSave({
```

Comment:
In the data frame analytics wizard, I am getting an error at the create index pattern step:

![image](https://user-images.githubusercontent.com/7405507/93775463-56925000-fc1a-11ea-9d86-4546e480c89a.png)